apps/web/src/store/modules/file/type.ts

```diff
@@ -17,3 +17,7 @@ export interface ExcelDataField {
     reference?: Reference;
     options?: Options;
 }
+
+export interface ExportResponse {
+    download_url: string;
+}
```

Comment:
suggestion: ExportResponse seems that it is the response from the backend API.
I think it makes more sense to exist in the file that includes related API models.